tests/integration/conftest.py

```diff
@@ -768,3 +768,37 @@ def is_adv_enabled(client) -> bool:
     query_str = "query IsAdvEnabledPyApi { user { isAdvEnabled } }"
     response = client.execute(query_str)
     return bool(response['user']['isAdvEnabled'])
+
+
+IMAGE_URL = "https://storage.googleapis.com/diagnostics-demo-data/coco/COCO_train2014_000000000034.jpg"
+EXTERNAL_ID = "my-image"
+
+
+@pytest.fixture
+def big_dataset(dataset: Dataset):
+    task = dataset.create_data_rows([
+        {
+            "row_data": IMAGE_URL,
+            "external_id": EXTERNAL_ID
+        },
+    ] * 3)
+    task.wait_till_done()
+
+    yield dataset
```

Comment:
I think you have just moved this code, but anyway, should now we delete these data rows after `yield`?

Reply:
They should be deleted as part of the teardown https://github.com/Labelbox/labelbox-python/blob/f0d5fe93e9b3d52efe532408dcbd4499425baf9a/tests/integration/conftest.py#L241